atramhasis/static/js/concept.js

```diff
@@ -0,0 +1,12 @@
+$(document).ready(function() {
+    var searchUrl = window.location.search.substr(1);
+    const params = searchUrl.split();
+
+    if (params.findIndex(element => element === 'view_tree') > -1) {
```

Comment:
me again: veel te fancy javascript hier voor een public frontend ze, we ondersteunen ie6 nog in de jinja's. best met indexof werken, en geen arrow functions. haal de wayback machine maar boven